reference/api/displayed_attributes.md

```diff
@@ -42,6 +42,8 @@ List the settings.
 
 Update the [displayed attributes](/learn/configuration/settings.md#displayed-attributes) of an index. The index [`uid`](/learn/core_concepts/indexes.md#index-uid) is required.
 
+If your dataset contains nested structures, you can use dot notation to set a nested field as a displayed attribute.
```

Comment:
I think you should link the article here and the rest of the API reference 
```suggestion
If your dataset contains [nested structures](/learn/advanced/datatypes.md#nested-objects), you can use dot notation to set a nested field as a displayed attribute.
```

Reply:
A couple of things.

First of all, I clarified some aspects of the spec with @gmourier yesterday. To keep things short, the behaviour described there by default concerns all objects in a document field in the sense that all document fields containing objects are flattened to the root level of the document. I have simplified the text in this paragraph to reflect that.

Second, dot notation is a fairly simple topic for most programmers (`keyA.keyB[.keyX.etc]`). The behaviour described in `datatypes.md` is super specific to Meilisearch's core engine, not always easy to understand, and unlikely to be relevant to the majority of our users. I would like to avoid connecting them because I'm afraid a user might think e.g. they need to account for the stuff we do to arrays of objects.